libjulia/backends/evm/AbstractAssembly.h

```diff
@@ -66,6 +69,21 @@ class AbstractAssembly
 	/// Append a reference to a to-be-linked symobl.
 	/// Currently, we assume that the value is always a 20 byte number.
 	virtual void appendLinkerSymbol(std::string const& _name) = 0;
+
+	/// Append a jump instruction.
+	/// @param _stackDiffAfter the stack adjustment after this instruction.
+	virtual void appendJump(int _stackDiffAfter) = 0;
+
+	/// Append a jump-to-immediate operation.
+	virtual void appendJumpTo(LabelID _label, int _stackDiffAfter = 0) = 0;
+	/// Append a jump-to-if-immediate operation.
+	virtual void appendJumpToIf(LabelID _label) = 0;
```

Comment:
No, because it has a continuing code path. Stack adjustment can only be done if you have non-continuing control-flow. We need this for stack height checking because we do not analyze the control flow.